src/pages/version/NameChangeModal/getFormSchema.ts

```diff
@@ -0,0 +1,28 @@
+import { SpruceFormProps } from "components/SpruceForm/types";
+import { LeafyGreenTextArea } from "components/SpruceForm/Widgets/LeafyGreenWidgets";
+
+export const getFormSchema = (
+  name: string
+): {
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  schema: {
+    type: "object" as "object",
+    properties: {
+      newPatchName: {
+        title: "New patch name",
```

Comment:
[nit] I think we capitalize each letter of the label, at least from referencing the Project Settings page!

```suggestion
        title: "New Patch Name",
```